### ISSUE TITLE
Parameter for commit author in git

### ISSUE THREAD

**Issue by martivo** [entigolabs/entigo-k8s-gitops#3]
Currently the author is hardcoded as "jenkins". Default value should be "entigo-k8s-gitops" and should be possible to set the e-mail and name with flags.
```
--git-authror-name=Martin --git-athor-email=martin.vool@entigo.com
```

**Comment by kertkukk** [comment]
Implemented in [1fa4e8e](https://github.com/entigolabs/entigo-k8s-gitops/commit/1fa4e8ef05142c91045aa718e36a12014d18b740) @martivo

**Comment by martivo** [comment]
Works!
